server/config/config.example.js

```diff
@@ -1,44 +0,0 @@
-const path = require('path');
```

Comment:
You accidentally removed `config.example.js`. Add it back in and make another commit.